js/classes/FlexibleMonthCalendar.js

```diff
@@ -230,7 +230,7 @@ class FlexibleMonthCalendar extends Calendar
 
         $('.waiver-trigger').off('click').on('click', function()
         {
-            const dayId = $(this).closest('tr').attr('id').substr(3);
+            const dayId = $(this)[0].nextSibling.nextSibling.nextSibling.nextSibling.id;
```

Comment:
With this solution, we end up being too restricted to the number of elements and their position. Something like the following should work:

```suggestion
            const dayId = $(this).siblings().closest('.time-cells').attr('id');
```

Can you see if this works?

Reply:
Hi @thamara, yeah you are right, your change work as well. I applied the change on the latest commit.